docs/tutorials/nas/enas_proxylessnas.md

```diff
@@ -142,6 +142,22 @@ The resulting architecture is:
 mynet.graph
 ```
 
+## Store the trained model as a static network
+
+The trained ENAS network can be saved to disk for future inferences.
+
+```{.python .input}
+mynet.export('enas')
+```
+
+Load it back with mxnet:
+
+```{.python .input}
+mynet.export('enas', 0)
```

Comment:
Does it matter what epoch you export?  I imagine you'd need to export the most recently trained epoch, epoch=1 to get the same results as what we just got.  Is that right?

Reply:
the epoch is for the name appended to the model prefix, not really a constraint anyway

Reply:
Oh, so it doesn't change the contents of the exported file, just the file name?

Reply:
Yes, that is correct

Reply:
Will this confuse users?  Why not change the argument-name from `epoch` -> `name_prefix`?

Reply:
Will this confuse users? Maybe should clarify this in the documentation, or even rename `epoch` arg to something like `name_suffix`

Reply:
Actually, if it's only a suffix for a name and doesn't change the data saved in any way, then why have the epoch input at all?  Why not just have modelPathBase or modelDirectory as the parameter, and the user can choose what name prefix or directory (with fixed file names under it) to save the model in?  Seems more clear to me to save a model in a directory where I choose the directory name then to have to worry about the fact that it saves the model as 2 files and needs to be loaded as 2 seperate files too.  Just my two cents.

Reply:
Well, it's somewhat a mxnet's convention that models are saved with four digit epoch number for identification. I agree that in autogluon user doesnot necessarily have to care about that. Changed in the updated version.